jsk_perception/CMakeLists.txt

```diff
@@ -258,6 +260,7 @@ add_library(${PROJECT_NAME} SHARED ${jsk_perception_nodelet_sources}
   slic/slic.cpp
   src/histogram_of_oriented_gradients.cpp
   )
+add_darknet_definitions(${PROJECT_NAME})
```

Comment:
Could you tell me and write comment why this is necessary? Because this macro/function is not standard and I/others can't understand why this is necessary here.